daemon/stop.go

```diff
@@ -89,3 +90,21 @@ func (daemon *Daemon) StopPodWithinLock(pod *Pod) (int, string, error) {
 
 	return vmResponse.Code, vmResponse.Cause, nil
 }
+
+func (daemon *Daemon) StopContainer(container string) error {
+	p, idx, ok := daemon.PodList.GetByContainerIdOrName(container)
+	if !ok {
+		return fmt.Errorf("can not found container %s", container)
+	}
+
+	containerId := p.status.Containers[idx].Id
+	glog.V(1).Infof("found container %s to stop", containerId)
+
+	err := p.vm.KillContainer(containerId, syscall.SIGTERM)
+	if err == nil {
+		return nil
+	}
```

Comment:
p.vm.KillContainer(containerId, syscall.SIGTERM) hardly fail. but the process may continue to run.
please wait/check the container's status(we haven't had such api) in several seconds and then kill SIGKILL if needed.
or directly kill SIGKILL.